### PR TITLE
Enable simple travis CI which install tarantool-graphql and run tests

### PR DESCRIPTION
**Pull request by SudoBobo**
Related to #27

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  - cd graphql
+  - git submodule update --recursive --init
+  - tarantoolctl rocks install lulpeg
+  - tarantoolctl rocks install lrexlib-pcre
+  - cd ..
+  - sudo apt-get install lua5.1
+  - sudo apt-get install liblua5.1-0-dev
+  - wget "http://luarocks.github.io/luarocks/releases/luarocks-2.4.4.tar.gz"
+  - tar xf luarocks-2.4.4.tar.gz
+  - ls
```
Comment (Totktonada) [inline .travis.yml]: Don’t forget to remove.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+language: c
```
Comment (Totktonada) [inline .travis.yml:1]: It is needed?

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+sudo: true
+
+dist: trusty
+
+branches:
+  only:
+    - master
+    - sb/travis-ci
```
Comment (Totktonada) [inline .travis.yml]: PRs will be checked?

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  - cd luarocks-2.4.4
+  - ./configure
+  - make build
+  - sudo make install
+  - cd ../graphql
+  - sudo luarocks install luacheck
+  - sudo pip install virtualenv
+
+script:
+  - make test
```
Comment (Totktonada) [inline .travis.yml]: No newline symbol in the last line.

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  - echo "deb http://download.tarantool.org/tarantool/1.9/ubuntu/ trusty main" |
+    sudo tee /etc/apt/sources.list.d/tarantool_1_9.list
+  - echo "deb-src http://download.tarantool.org/tarantool/1.9/ubuntu/ trusty main" |
+    sudo tee -a /etc/apt/sources.list.d/tarantool_1_9.list
+
+install:
+  - sudo apt-get update
+  - sudo apt-get -y install tarantool tarantool-dev
+  - cd ..
+  - git clone https://github.com/rtsisyk/msgpuck
```
Comment (Totktonada) [inline .travis.yml:30]: Are there problems with https://launchpad.net/ubuntu/+source/msgpuck ?

File: .travis.yml [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+  - sudo make install
+  - cd ..
+  - tarantoolctl rocks install shard
+  - tarantoolctl rocks install avro-schema
+  - cd graphql
+  - git submodule update --recursive --init
+  - tarantoolctl rocks install lulpeg
+  - tarantoolctl rocks install lrexlib-pcre
+  - cd ..
+  - sudo apt-get install lua5.1
```
Comment (Totktonada) [inline .travis.yml]: Please, comment that lua is needed for luacheck, create the issue in https://github.com/tarantool/rocks and add issue number here.

File: .travis.yml [hunk 7]
```diff
@@ -0,0 +1,57 @@
+language: c
+
+sudo: true
+
+dist: trusty
```
Comment (Totktonada) [inline .travis.yml:5]: Why trusty? There is more fresh LTS: xenial.

**Review by Totktonada — COMMENTED** [review]
See comments below.

File: .travis.yml (hunk 2 above)
Comment (SudoBobo) [reply .travis.yml:1]: Yes, it installs a big part of environment; also it's a common way to build an environment while working with Lua (as travis does not naturally support `language: Lua`)

File: .travis.yml (hunk 3 above)
Comment (SudoBobo) [reply .travis.yml]: Yes, travis builds all PRs by default. And it uses .travis.yml from the branch you want with master.

Comment (SudoBobo) [reply .travis.yml]: Not sure for which branches we should use travis builds on every commit (except master)

File: .travis.yml (hunk 4 above)
Comment (SudoBobo) [reply .travis.yml]: fixed.

File: .travis.yml (hunk 1 above)
Comment (SudoBobo) [reply .travis.yml]: fixed.

File: .travis.yml (hunk 7 above)
Comment (SudoBobo) [reply .travis.yml:5]: Not currently supported by travis. But we can use Docker inside travis environment to run our tests on xenial.

File: .travis.yml (hunk 5 above)
Comment (SudoBobo) [reply .travis.yml:30]: `sudo apt-get install libmsgpuck-dev` failed, package was not found.

File: .travis.yml (hunk 3 above)
Comment (Totktonada) [reply .travis.yml]: Just don’t forget to remove 'sb/travis-ci' before merge into master.

File: .travis.yml (hunk 5 above)
Comment (Totktonada) [reply .travis.yml:30]: It seems that libmsgpuck-dev was built only for yakkety. Okay.

File: .travis.yml (hunk 7 above)
Comment (Totktonada) [reply .travis.yml:5]: There is the reason, so okay.

File: .travis.yml (hunk 6 above)
Comment (SudoBobo) [reply .travis.yml]: https://github.com/tarantool/rocks/issues/21

Comment (Totktonada) [reply .travis.yml]: In-source comment?

**Review by Totktonada — APPROVED** [review]
LGTM. Please add the rockspeck issue link to a comment and merge it.

**Pull request merged**